config/config.php

```diff
@@ -326,3 +326,10 @@ functionality to PhpReport
  * To be used in combination with ISSUE_TRACKER_LINKS_URL, see its entry above.
  */
 define('ISSUE_TRACKER_LINKS_TEXT', serialize(array('Report an issue')));
+
+/* CalDav integration */
+define('CALENDAR_URL', '');
+define('CALENDAR_ID', '');
+define('CALENDAR_USERNAME', '');
+define('CALENDAR_PASSWORD', '');
+define('COMPANY_DOMAIN', '');
```

Comment:
We also have `define('NO_FILL_EMAIL_DOMAIN', "domain.com");` for a similar purpose: compose email addresses for our users.

The name of the existing parameter is quite specific, maybe we could make the other feature use `COMPANY_DOMAIN`? Or keep both, no strong opinion. What we would really need is to work on #434...

Reply:
Oh #434 looks really useful indeed, I'll try to prioritize it. I renamend `NO_FILL_EMAIL_DOMAIN` to `COMPANY_DOMAIN`